airbyte-oauth/src/main/java/io/airbyte/oauth/flows/LeverOAuthFlow.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.oauth.flows;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.oauth.BaseOAuthFlow;
+import org.apache.http.client.utils.URIBuilder;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Supplier;
+
+public class LeverOAuthFlow extends BaseOAuthFlow {
+
+  private static final String AUTHORIZE_URL = "https://sandbox-lever.auth0.com/authorize";
+  private static final String ACCESS_TOKEN_URL = "https://sandbox-lever.auth0.com/oauth/token";
+  // private static final String ACCESS_TOKEN_URL = "https://api.sandbox.lever.co/oauth/token";
+  private static final String SCOPES = String.join("+", "applications:read:admin",
+      "contact:read:admin",
+      "interviews:read:admin",
+      "offers:read:admin",
+      "opportunities:read:admin",
+      "postings:read:admin",
+      "referrals:read:admin",
+      "requisitions:read:admin",
+      "resumes:read:admin",
+      "sources:read:admin",
+      "stages:read:admin",
+      "offline_access");
+
+  private String getAudience() {
+    return "https://api.sandbox.lever.co/v1/";
```

Comment:
Sandbox

Reply:
See other comment. Lever's application approval is needed.

Reply:
fixed